packages/larva-css/src/03-utilities/u-color.common.async.scss

```diff
@@ -26,3 +26,8 @@ $colors: (
 .lrv-u-color-grey-dark\:hover:hover {
 	color: map-get( $TOKENS-MAP, color-grey-dark );
 }
+
+// TODO: This value should be moved to come from a map in settings
+.lrv-u-color-brand-primary-dark\:hover:hover {
```

Comment:
@yahilmadakiya Also need to do npm run build so that the styles compile

Reply:
Done